src/sql/azdata.proposed.d.ts

```diff
@@ -563,11 +563,13 @@ declare module 'azdata' {
 	}
 
 	export enum DataProviderType {
-		TableDesignerProvider = 'TableDesignerProvider'
+		TableDesignerProvider = 'TableDesignerProvider',
+		ExecutionPlanServiceProvider = 'ExecutionPlanServiceProvider'
```

Comment:
ExecutionPlanProvider or ExecutionPlanServiceProvider?

Reply:
I think we've generally just use the term `Provider`